src/Generate.svelte

```diff
@@ -82,8 +82,8 @@
   $: numPromptTokens = Math.ceil(prompt.length / 4) // very rough estimation https://www.reddit.com/r/StableDiffusion/comments/wl4cn3/the_maximum_usable_length_of_a_stable_diffusion/
 
   $: {
-    stableDiffusionCommand = `python scripts/txt2img.py --prompt "${prompt}" --plms --n_samples ${samples?.toString()} --scale ${scale?.toString()} --n_iter ${iter?.toString()} --ddim_steps ${steps?.toString()} --H ${height?.toString()} --W ${width?.toString()} --seed ${seed?.toString()}`
-    stableDiffusionCommandHtml = `python scripts/txt2img.py --prompt <strong>"${prompt}"</strong> --plms --n_samples <strong>${samples}</strong> --scale <strong>${scale}</strong> --n_iter <strong>${iter}</strong> --ddim_steps <strong>${steps}</strong> --H <strong>${height}</strong> --W <strong>${width}</strong> --seed <strong>${seed}</strong>`
+    stableDiffusionCommand = `python scripts/txt2img.py --prompt "${prompt}" --plms --n_samples ${samples?.toString()} --scale ${scale?.toString()} --n_iter ${iter?.toString()} --ddim_steps ${steps?.toString()} --H ${height?.toString()} --W ${width?.toString()} --seed ${seed?.toString()} --fixed_code`
+    stableDiffusionCommandHtml = `python scripts/txt2img.py --prompt <strong>"${prompt}"</strong> --plms --n_samples <strong>${samples}</strong> --scale <strong>${scale}</strong> --n_iter <strong>${iter}</strong> --ddim_steps <strong>${steps}</strong> --H <strong>${height}</strong> --W <strong>${width}</strong> --seed <strong>${seed}</strong> --fixed_code`
```

Comment:
this seems to make images reproduceable

Reply:
Cool! It really works. There's a catch though. Images generated previously (before `--fixed_code`) are not repeatable.

I wonder if we want to make this a toggle. The only issue is that if you now generate multiple copies without checking Random, it will produce the same image.

Reply:
the way that stablediffusion does it: if multiple copies AND random, then each copy increments seed by 1

Reply:
Which SD parameter do you associate with "multiple copies"? `--n_iter` stacks multiple copies vertically in the same image.

My implementation of multiple copies (the 1-10 dropdown next to the Generate button) is independent of that. It simply runs the same command X times, in sequence. So now with `--fixed_code` it produces the same image for every copy, unless Random is checked.

I'm thinking to disable the copies dropdown (and reset it to 1) when Random is checked, enable it when Random is unchecked.

Reply:
SD parameter is `--n_samples`

but for our implementation we could do if Random checked AND multiple copies then we just increment the seed of each by one (since they are now deterministic). this is useful for just generating a bunch of stuff in case we get a bad seed

Reply:
OK so here's what I've decided. Right now I'm in the middle of implementing parametric prompts (or prompt matrix). Once I finish I want to:

- re-enable `n_samples` and name it Batch Size.
- put a warning next to the help bubble to warn against crashing and/or high mem usage. Strange thing is that for me it only crashes when I set it to 2. Other values work fine. I tried up to 4. More than that doesn't seem to make sense, since it produces n variations in the same image.
- disable the copies dropdown when either Random is checked, or the seed is -1